docs/source/conf.py

```diff
@@ -75,6 +75,7 @@
     "inherited-members": True,
     "member-order": "bysource",
     "autosummary": True,
+    "exclude-members": "objects",
```

Comment:
@swheaton one alternative here would be to exclude `mongoengine` subpackages altogether:
https://github.com/rdb/sphinx-autopackagesummary/issues/4#issuecomment-630211238

This would be nice because we intend to remove it as a dependency and therefore any inherited members really should not be documented anyway.

Also slight benefit that we can have `objects` members on classes if we want 🤓

Reply:
oh this is a different package than `sphinx.ext.autodoc`

Reply:
ah it is available for autodoc as well:
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports

```
autodoc_mock_imports = ["mongoengine"]
```

Reply:
tried it already, doesn't work, even if I try `fiftyone.core.odm.dataset.mongoengine` :(

When I researched why, I think I found that it only mocks if the package is NOT available, otherwise uses the available package. Which is not what we want in our case.

Reply:
Even if you tell it `inherited-members=False`, it still tries to access inherited members ...